election/serializers.py

```diff
@@ -25,11 +25,11 @@ class ElectionCreateSerializer(ElectionViewMixin, serializers.ModelSerializer):
     def create(self, data):
         # Copy the validated_data
         validated_data = dict(data)
-        validated_data["on_invitation_only"] = False
         if "elector_emails" in validated_data:
-            if validated_data["elector_emails"] != []:
-                validated_data["on_invitation_only"] = True
             validated_data.pop("elector_emails")

```

Comment:
you removed a special case: if the front sends an empty list of mails, it should be the same as not sending emails at all. Please revert;